### ISSUE TITLE
Does not download epub of calibre [en/ru]

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#442]
```
What steps will reproduce the problem?
1.push the button the network library 
(Нажимаю кнопку сетевой библиотеки)

2.choose customized opds source calibre library 
(Выбираю настроенный до этого opds источник 
библиотеки calibre)

3.choose from a list of book epub format
(выбираю из списка книгу формата epub)

4.box is displayed on "download the book ..."
(появляется окно "Загрузка книги...")

5.I click "OK"
(нажимаю кнопку "Ок")

What is the expected output?
Download this book
Загрузка книги


What do you see instead?
Appears "Error OPDS" - "java.lang.NullPointerException"
Появляется "Ошибка OPDS" - "java.lang.NullPointerException"


What version of the product are you using?
EbookDroid - v 1.6.4
Calibre - 0.9.17 and previous


On what operating system?
Android 4.0.3


What device are you using?
Sony Tablet S


Please provide any additional information below.
books of other formats with this calibre library loaded normally
книги других форматов с этой библиотеки 
calibre загружаются нормально
```

Original issue reported on code.google.com by `observer...@gmail.com` on 2 Feb 2013 at 5:18

**Comment by GoogleCodeExporter** [comment]
```
з.ы. хотел бы поддержать проект покупкой 
платной версии, только не могу понять - есть 
там русский интерфейс или нет?
```

Original comment by `observer...@gmail.com` on 3 Feb 2013 at 5:05

**Comment by GoogleCodeExporter** [comment]
```
Там 2 языка: русский и английский. Но там нет 
opds и вообще чего-либо связанного с сетью.
```

Original comment by `Andrei.K...@gmail.com` on 3 Feb 2013 at 8:19

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 1.6.5+
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 5 Feb 2013 at 7:06
- Changed state: **Fixed**
